test/set/edge-cases.spec.js

```diff
@@ -42,7 +42,7 @@ describe("Special Cases", function() {
         var edgeCaseCache = {
             jsonGraph: {
                 user: {
-                    name: "Jim",
+                    name: {$type: $atom, value: "Jim"},
```

Comment:
This test was actually testing for buggy behavior before. It was asserting that the name field wasn't getting wrapped, which was wrong.